docs/blog/2018-11-29-gatsby-analogy/index.md

```diff
@@ -0,0 +1,92 @@
+
+
+![casper mattress inflating](./images/casper-unboxing.gif)
+
+**Both ship small and bloom into much more.**
+
+## 4. Prefetch resources for snappy routing
+
+It’s not just important that Gatsby sites _are_ fast. It’s also important that they _seem_ even faster.
+
+When a Gatsby page finishes loading, Gatsby starts prefetching resources for internal pages the user may navigate to (internal links that are visible in the viewport). It does this at a very low priority -- it will prioritize almost anything else for the current page over these prefetching requests. 
+
+Additionally, when a link is actively hovered over (a strong indication the user will visit that link), Gatsby will fetch the resources for that page at a higher priority.  
+
+This means that when the user navigates to another internal page, resources have already been quietly loaded in the background and routing feels instantaneous.
+
+Say you order delivery for dinner. It arrives, you eat. Later, you look at the menu again and scan for dessert. You decide on one and press “order”. Your doorbell rings immediately, your dessert has arrived. It’s like the delivery person is a psychic. They arrive instantaneously, with exactly what you wanted, exactly when you want it.
+
+![the flash delivering pizza](./images/the-flash-pizza.gif)
+
+**Gatsby preloads resources for pages a user is likely to navigate to, like a psychic delivery person.**
+
+## 5. Consolidate data sourcing
+
+With Gatsby, your data can be sourced from just about anywhere -- content management systems, markdown, APIs, databases, etc. A recent series on the Gatsby blog detailed the concept of the “[content mesh](https://www.gatsbyjs.org/blog/2018-10-04-journey-to-the-content-mesh/)”:
+
+> [Gatsby is] the infrastructure layer for a decoupled website. The content mesh stitches together content systems in a modern development environment while optimizing website delivery for performance.
+
+With Gatsby, you can source your data into a consolidated data layer, which you can query against directly. No matter where your data comes from, you can access it all from a single query.
+
+This is like ordering from Amazon. Instead of heading to the grocery store, the hardware store, the electronics store, the pet store, etc., you can place a single order to Amazon to get items from them all. You don't have to worry about logistics, combining data, or managing multiple requests — it Just Works™.
+
+You can make a single order to Amazon for a variety of items originating from a variety of sources; You can make a single query to the Gatsby data layer for a range of data originating from many different data sources.
+
+**Gatsby consolidates your data into a single, queryable layer, like Amazon consolidates the online ordering experience from a multitude of vendors.**
+
+## Wrap up
+
+These are a bit silly, but I hope you enjoyed and found this helpful!
```

Comment:
I don't think these are that silly! I think they're actually really helpful :)

I would possibly refine this conclusion a bit more, but otherwise I think this all looks great!

Reply:
Fleshed it out a bit more!